Automattic/simplenote-android
# Fastlane setup for local screenshot automation

### Fix
Sets up the components required for running the screenshots automation locally, that is integrates `screengrab` in the projects and adds a new lane to the `Fastfile` for it.

### Test
1. Launch an Emulator (I've been using a Pixel 2 XL with API 29)
1. If you already have Simplenote on the Emulator, please logout. The full automation will take care of this.
1. Run `bundle install && bundle exec fastlane take_screenshots`

When everything is done, your browser should open rendering the content of `fastlane/screenshots/screenshots.html`, which is a screenshot of the login screen.

<img width="1544" alt="Screen Shot 2020-05-12 at 10 08 05 pm" src="https://user-images.githubusercontent.com/1218433/81688697-161d0b80-949d-11ea-8bb7-ce3ceb3e73a8.png">


The next step will be adding all of the code to the screenshot test to take the ones we need.

### Review
Only one developer is required to review these changes, but anyone can perform the review. Thanks.

### Release
These changes do not require release notes.
